packages/theme/src/lib/light-theme.ts

```diff
@@ -1559,6 +1560,11 @@ export const lightTheme = {
       defaultProps: {
         variant: 'outlined',
       },
+      styleOverrides: {
+        root: {
+          borderColor: tokens.borderDecorative,
```

Comment:
The color was correct. My understanding was that we were trying to ensure the token was being used.

Reply:
ahh. it's just making sure there's no remaining misuses of `backgroundBorder` when it should be using `borderDecorative` (since UX was misusing `backgroundBorder` in the specs). However, the divider color in the palette is using the `borderDecorative` token and that is what determines the border color, so we're good on that front.